### PR TITLE
Update the license so that @mageekguy is not the only copyright holder

### PR DESCRIPTION
**Pull request by jubianchi**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Copyright (c) 2016, atoum contributors.
```
Comment (Hywan) [inline LICENSE]: Would it not be better to have `2016-2016` for a better scripting update?

File: README.md [hunk 2]
```diff
@@ -339,3 +339,7 @@ Looking for a roadmap?
 
 - [Here is](https://github.com/atoum/atoum/labels/In%20progress) the work in progress,
 - [And there](https://github.com/atoum/atoum/milestones/*) what will come in next releases.
+
+## License
+
+atoum is released under the BSD-3-Clause License. See the bundled [LICENSE](LICENSE) file for details.
```
Comment (Hywan) [inline README.md]: I guess that `[LICENSE]()` still works.

File: CREDITS.md [hunk 3]
```diff
@@ -1,11 +1,12 @@
-#Main contributors
+# Main contributors
 
+* Frédéric Hardy (frédéric.hardy@atoum.org)
 * Julien Bianchi (julien.bianchi@atoum.org)
 * Ludovic Fleury (ludovic.fleury@atoum.org)
 * Gérald Croes (gerald.croes@atoum.org)
 * François Dussert (francois.dussert@atoum.org)
```
Comment (Hywan) [inline CREDITS.md]: Is this list still up-to-date?
Does it make sense to keep it?

File: README.md (hunk 2 above)
Comment (jubianchi) [inline README.md]: Yes it should work in Github but to be sure it works with most markdown parsers, I prefer to keep this as is.

File: CREDITS.md (hunk 3 above)
Comment (jubianchi) [inline CREDITS.md]: I'll propose something else ;)

File: README.md [hunk 4]
```diff
@@ -339,3 +339,25 @@ Looking for a roadmap?
 
 - [Here is](https://github.com/atoum/atoum/labels/In%20progress) the work in progress,
 - [And there](https://github.com/atoum/atoum/milestones/*) what will come in next releases.
+
+## Credits
+
+atoum has been created by [Frédéric Hardy]. It is now led by a strong community of contributors. You can find them 
+in the [committer list](https://github.com/atoum/atoum/graphs/contributors) or in the 
+[Contributors team](https://github.com/orgs/atoum/teams/contributors).
```
Comment (Hywan) [inline README.md]: Is it voluntary to have a capital C to Contributors?

Comment (jubianchi) [inline README.md]: yes: the team name has a capital "C" ;)

File: LICENSE [hunk 5]
```diff
@@ -1,3 +1,4 @@
+Copyright (c) 2016-2016, atoum contributors.
 Copyright (c) 2010-2016, Frédéric Hardy.
```
Comment (mikaelrandy) [inline LICENSE]: 2 lines of Copyright ?

Comment (jubianchi) [inline LICENSE]: @mikaelrandy yes. If we were to change the license we would have to ask all contributor if they are OK with that.

Moreover, removing the name of @mageekguy from the license does not seem good to me: he was the copyright owner. We just choose to change this from now on.

**Pull request merged**